src/kitconcept/glossary/browser/views.py

```diff
@@ -175,7 +176,8 @@ def truncateDescription(self, text):
             text = text.strip()
             text = u"{0}{1}".format(text, ellipsis)
 
-        text = text.encode("utf-8", "replace")
```

Comment:
@rodfersou shouldn't we use encode here that works for both py2 and py3?

Reply:
the encode works, but later we try to do a replace that don't works.

https://github.com/kitconcept/kitconcept.glossary/blob/679a41529325dede45064657d7d67b42854ce4a4/src/kitconcept/glossary/browser/views.py#L189

I'll add a test to show better the problem after make the buildout green again on master.

Reply:
@tisto here I prove that it is broken for Python 3:
https://travis-ci.org/github/kitconcept/kitconcept.glossary/jobs/673332140